.github/ISSUE_TEMPLATE/1.bug.yml

```
@@ -9,7 +9,7 @@ body:
     attributes:
       label: Search keywords
       description: |
-        Your issue may have already been reported! First search for duplicates among the [existing issues](https://github.com/mui/mui-x/issues).
+        Your issue may have already been reported! First search for duplicates among the [existing issues](https://github.com/mui/mui-x/issues?q=is%3Aissue+).
```

Should it return PRs?

```suggestion
        Your issue may have already been reported! First search for duplicates among the [existing issues](https://github.com/mui/mui-x/issues?q=).
```

I guess not

---

not sure ... could be helpful, but also misleading. 

What we could also do is link the github search page, that is way more powerful, so we can also apply a filter by default that searches for issues in the whole org and not only on mui-x.

This is an [example](https://github.com/search?q=org%3Amui+is%3Aissue&type=issues) of that

WDYT?

---

But to be consistent with https://mui.com/x/introduction/support/#github

---

makes sense ... I will adjust the link then

---

> This is an [example](https://github.com/search?q=org%3Amui+is%3Aissue&type=issues) of that

This page feels too broad IMHO

> not sure ... could be helpful, but also misleading.

Yeah, no preferences on including PRs